frame_id: odom_combined
detect_continuous: true
crop_box: [0.0, 2.0, -1.0, 1.0, 0.1, 1.0]
up_vector: [0.0, 0.0, 1.0]
up_vector_thresh: 0.087
inlier_thresh: 0.02
min_cluster_size: 100
cluster_tolerance: 0.04